app/commands/git/sync_concept_exercise.rb

```diff
@@ -19,11 +19,30 @@ def call
         taught_concepts: find_concepts(exercise_config[:concepts]),
         prerequisites: find_concepts(exercise_config[:prerequisites])
       )
+
+      update_authors!
+      update_contributors!
     end
 
     private
     attr_reader :exercise
 
+    def update_authors!
+      authors = ::User.where(handle: author_usernames_config)
+      authors.each { |author| ::Exercise::Authorship::Create.(exercise, author) }
```

Comment:
```suggestion
      authors.find_each { |author| ::Exercise::Authorship::Create.(exercise, author) }
```

Ok yes, it's _unlikely_ that we'll hit > 500 records here, but good practice to use `find_each` over `each when you can.

- `each` loads everything into memory
- `find_each` batches

Reply:
Excellent.